tests/testflows/map_type/configs/clickhouse/config.d/ports.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<yandex>
+    <https_port>8443</https_port>
```

Comment:
Why do we need TLS to test Map data type?

Reply:
Removed.